Week01/Problem05/klai_05.py

```diff
@@ -0,0 +1,7 @@
+#import euler3 as el
+from operator import mul
+
+primes = [2,2,2,2,3,3,5,7,11,13,17,19]
+print reduce(mul, primes, 1)
```

Comment:
Nice use of built-in functions and existing modules.  Using existing infrastructure will often save hours, if not days, of time spent trying to program & debug a particular subroutine from scratch.